src/tools.rs

```diff
@@ -1039,9 +1035,8 @@ DKIM Results: Passed=true, Works=true, Allow_Keychange=true";
 
     #[test]
     fn test_create_outgoing_rfc724_mid() {
-        let mid = create_outgoing_rfc724_mid("foo@bar.de");
+        let mid = create_outgoing_rfc724_mid();
         assert!(mid.starts_with("Mr."));
-        assert!(mid.ends_with("bar.de"));
```

Comment:
Maybe test for "@localhost" instead of just removing? Otherwise the fix can be reverted, having only `test_prepare_not_configured()` is a weak defence